guides/user/managed-delivery/resource-status/index.md

```diff
@@ -128,18 +128,32 @@ When you resume management the status will change to [Resumed](/guides/user/mana
   image_path="./resource-status-resumed.png"
 %}
 
-When a resource has a Resumed status, it means management was just resumed after being temporarily paused and Spinnaker hasn't checked for drift from the declarative configuration yet.
+When a resource has this status, it means management was just resumed after being temporarily paused and Spinnaker hasn't checked for drift from the declarative configuration yet.
 
 If a drift is detected, the status will change to [Diff](/guides/user/managed-delivery/resource-status/#diff). If no drift is detected, the status will change to [Happy](/guides/user/managed-delivery/resource-status/#unhappy).
 
+### Currently unresolvable
+{%
+  include
+  figure
+  image_path="./resource-status-currently-unresolvable.png"
+%}
+
+When a resource has this status, it means something Spinnaker requires to properly manage the resource is either not ready yet or temporarily experiencing disruption. Automatic action won't be taken while in this status, but it doesn't necessarily mean something is broken. Some examples of things that might lead to a resource entering this status:
+
+  - A downstream system (like a container registry or cloud provider) temporarily rate limiting or experiencing availability issues
+  - A required part of the resource's desired state (like a VM image) not being available yet
+
+If you see this status and want to investigate the cause, you can click "History" to view additional details and understand what else was happening when the issue began. While you may decide to take some action as a result of this status, it's unlikely the issue requires manual intervention. Once the issue is resolved the resource will transition to whatever status is applicable at the time based on its desired state.
```

Comment:
can we add a screenshot/link when we send the user to check the `history`?

Reply:
you mean like a picture of what the history event looks like when a resource is in this status? If so, I have plans for that! I'm thinking about having a separate doc page about how to use the history view with a guide similar to this one covering each history event type — once that's added we can link between this page and that page so folks can understand how the two connect to each other.

Reply:
Sounds AMAZING! can't wait ❤️